frontend/src/concepts/pipelines/elyra/utils.ts

```diff
@@ -105,3 +118,49 @@ export const generateElyraServiceAccountRoleBinding = (
     },
   ],
 });
+
+export const createElyraServiceAccountRoleBinding = async (
+  notebook: NotebookKind,
+): Promise<RoleBindingKind | void> => {
+  const notebookName = notebook.metadata.name;
+  const namespace = notebook.metadata.namespace;
+  const notebookUid = notebook.metadata.uid;
+
+  // Check if rolebinding is already exists for backward compatibility
+  const roleBinding = await getRoleBinding(
+    namespace,
+    getElyraServiceAccountRoleBindingName(notebookName),
+  ).catch((e) => {
+    // 404 is not an error
+    if (e.statusObject?.code === 404) {
+      return undefined;
+    }
+    throw e;
+  });
+
+  if (roleBinding) {
+    const ownerReferences = roleBinding.metadata.ownerReferences || [];
+    if (!ownerReferences.find((ownerReference) => ownerReference.uid === notebookUid)) {
+      ownerReferences.push(getElyraRoleBindingOwnerRef(notebookName, notebookUid));
+    }
+    return patchRoleBindingOwnerRef(
+      roleBinding.metadata.name,
+      roleBinding.metadata.namespace,
+      ownerReferences,
+    ).catch((e) => {
+      // This is not ideal, but it shouldn't impact the starting of the notebook. Let us log it, and mute the error
+      // eslint-disable-next-line no-console
+      console.error(
+        `Could not patch rolebinding to service account for notebook, ${notebookName}; Reason ${e.message}`,
+      );
+    });
+  }
+  return createRoleBinding(
+    generateElyraServiceAccountRoleBinding(notebookName, namespace, notebookUid),
+  ).catch((e) => {
+    // eslint-disable-next-line no-console
+    console.error(
+      `Could not create rolebinding to service account for notebook, ${notebookName}; Reason ${e.message}`,
+    );
+  });
+};
```

Comment:
Failing to create the role binding is ok?
This error will be suppressed and never shown to the user.

Reply:
https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/api/k8s/notebooks.ts#L226-L232 this is what the current behavior is, we don't show anything to the user.